jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -2214,7 +2225,7 @@ public VarHandle unreflectVarHandle(Field field) throws IllegalAccessException {
 				newPrevAccessClass = null;
 			}
 			
-			return new Lookup(accessClass, newPrevAccessClass, newAccessMode);
+			return new Lookup(accessClass, newPrevAccessClass, newAccessMode, true);
```

Comment:
Shouldn't this be false?  If the user already has a Lookup in the `java.lang.invoke` package, why would we restrict dropping access?

Reply:
`new Lookup(accessClass, newPrevAccessClass, newAccessMode)`  invokes `this(lookupClass, prevLookupClass, lookupMode, true)`, so the flag was `true` in current code base, i.e., always check the package.
This check matches RI behaviours which was introduced for defense-in-depth fix [1], and still presents in JDK15.

[1] http://mail.openjdk.java.net/pipermail/jigsaw-dev/2017-April/012268.html

Reply:
It's an odd restriction to leave at this point but as it matches the RI, we'll do the same for 15.